artifacts/flagger/crd.yaml

```diff
@@ -1180,19 +1180,16 @@ spec:
             spec:
               description: AlertProviderSpec defines the desired state of a AlertProvider.
               type: object
-              oneOf:
-                - required:
-                    - type
-                    - address
-                - required:
-                    - type
-                    - secretRef
+              required:
+                - type
+                - address
```

Comment:
@stefanprodan Not sure if there is a better alternative to schema validation than this change, but now address is configurable.

Reply:
Please undo this change as it breaks everyone’s config.